### ISSUE TITLE
VoiceOver improvements

### ISSUE THREAD

**Issue by BasThomas** [duckduckgo/iOS#676]
While using the app I found some places where VoiceOver can either use some more information, or where a page isn't accessible at all using VoiceOver.

- Changing the app icon is currently not accessible through VoiceOver
- The "Add to Dock" modal has an awkward naming for its close button ("remove" instead of "close".
- Consider adding an accessibility hint to the "tab toggle" to announce which option is currently selected — or add to the label, as hints can be disabled.

**Comment by brindy** [comment]
These are merged, thanks @BasThomas

**Comment by BasThomas** [comment]
The tab toggle accessibility hint hasn't been added yet, actually. Up to you if that's something you'd want to add, though :)

**Comment by brindy** [comment]
Ah, sorry - thanks!

**Comment by brindy** [comment]
I've added the accessibility hint for the toggle on my current branch which is WIP but hopefully going out soon.  Thanks again for the heads up and contributions.
